untraced/ykpack/src/build.rs

```diff
@@ -104,13 +98,13 @@ impl SirBuilder {
     /// Appends a statement to the specified basic block.
     pub fn push_stmt(&mut self, bb: BasicBlockIndex, stmt: Statement) {
         self.func.blocks[usize::try_from(bb).unwrap()]
-            .stmts
+            .mut_stmts()
```

Comment:
Would `stmts_mut()` be more idiomatic? E.g. `null_mut()`.

Reply:
Good point. Fixed in https://github.com/softdevteam/yk/pull/317/commits/23f0d81fbc410d9828411328b66e64002acb226b.